docs/changes/2625.api.rst

```diff
@@ -0,0 +1,8 @@
+Add possibility to use ``HIPPARCOS`` catalog to get star positions
+
+- add catalogs enumerator to ``ctapipe.utils.astro``
+- update ``get_bright_stars`` in ``ctapipe.utils.astro`` to allow catalog selection
+- ensure application of proper motion
+- add possibility to select stars on proximit to a given position and on apparent magnitude
+- move cached star catalogs to ``ctapipe/resources/``
+- API change: ``ctapipe.utils.astro.get_star_catalog`` now requires to give a time to apply proper motion, before there was no time argument
```

Comment:
This doesn't seem to be the case. The function is defined below as:
```py3
def get_star_catalog(
    catalog: str | StarCatalog, magnitude_cutoff: float = 8.0, row_limit: int = 1000000
) -> Table:
```

Reply:
@kosack yes, you're right. Fixed now.